deckhand/engine/schema/v1_0/default_schema.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+substitution_schema = {
```

Comment:
It's more intuitive for me that the schemas would be defined as non-code files.  I think either JSON or YAML would be reasonable.

Then we can specify it completely and include, e.g. the schema header: `"$schema": "http://json-schema.org/schema#"`